index.js

```diff
@@ -0,0 +1,248 @@
+},
+{
+  Linkk: 'See Project',
+},
+{
+  Titled1: 'Data Dashboard Healthcare',
+  Titlew1: 'Website Protfolio',
+  Titlepro: 'Profesional Art Printing Data More',
+  Description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has  been the industrys standard',
+  Tecnologies: ['HTML', 'bootstrap', 'Ruby'],
+  Link: 'See Project',
+}];
+
+//  Project Section Card creator
+const cardContainer1 = document.querySelector('#carddss');
+const cardContainer = document.querySelector('#section-02');
+cardContainer1.innerHTML += `<div class="h21-line">
+<h1 class="h21">${ProjectsMobile[0].Title1}</h1>
+<hr class="line">
+</div>
+<img class="main-image" src="${ProjectsMobile[0].Image}" alt="main image">
+<div class="card-1">
+<h2 class="multi-story">${ProjectsMobile[0].Title2}</h2>
+<p class="p21">${ProjectsMobile[0].Description}</p>
+<ul class="skills">
+<li class="css-1 button">${ProjectsMobile[0].Tecnologies[0]}</li>
+<li class="skill button">${ProjectsMobile[0].Tecnologies[1]}</li>
+<li class="skill button">${ProjectsMobile[0].Tecnologies[2]}</li>
+<li class="skill button">${ProjectsMobile[0].Tecnologies[3]}</li>
+</ul>
+</div>
+<button class ="see-project-1" id="seeprojectt" onclick="popupp()">${ProjectsMobile[0].Link}</button>
+`;
+
+for (let i = 0; i < 6; i++) {
+  const projectM = `<div class="card-2">
+<h2 class="prof-art">${ProjectsMobile[1].Name} </h2>
+<p class="p22">${ProjectsMobile[1].Description}</p>
+<ul class="skils">
+<li class="button">${ProjectsMobile[1].Tecnologies[0]}</li>
+<li class="skil button">${ProjectsMobile[1].Tecnologies[1]}</li>
+<li class="skil button">${ProjectsMobile[1].Tecnologies[2]}</li>
+</ul>
+</div>
+<button class ="see-project-2" id="seeproject${[1]}" onclick="popupp()">${ProjectsMobile[1].Link}</button>
+`;
+  cardContainer.innerHTML += projectM;
+}
+
+// Get the popup mobile
+const modal = document.getElementById('myModal');
+modal.innerHTML = `<div class="modal-content">
+<div class="modalHead-close">
+<h2 class = "modalHeader" id="modalHeader">${ProjectsMobile[2].Title1}</h2>
+<span class="close">${ProjectsMobile[2].Close}</span>
+</div>
+<div>
+<ul id="modalTech" class ="modalTech">
+  <li>${ProjectsMobile[2].Tecnologies[0]}</li>
+  <li>${ProjectsMobile[2].Tecnologies[1]}</li>
+  <li>${ProjectsMobile[2].Tecnologies[2]}</li>
+</ul>
+<img src="${ProjectsMobile[2].Image[0]}" alt="${ProjectsMobile[2].Image[1]}">
+<p class = "modal-pm" >${ProjectsMobile[2].Description}</p>
+<div class = "live-source">
+<a href="${ProjectsMobile[2].Links[0][0]}">${ProjectsMobile[2].Links[0][1]}<img src="${ProjectsMobile[2].Links[0][2]}" alt="${ProjectsMobile[2].Links[0][3]}"></a>
+<a href="${ProjectsMobile[2].Links[1][0]}">${ProjectsMobile[2].Links[1][1]} <img src="${ProjectsMobile[2].Links[1][2]}" alt="${ProjectsMobile[2].Links[1][1]}"></a>
+</div>
+</div>
+</div>`;
```

Comment:
- [ ] As mentioned by the first reviewer, kindly make sure the popup modal for each project is populated with the data from the projects array and not manually like the above. Right now you have populated the same popup modal for all the projects. Please make sure each popup modal is populated with its project details.